### PR TITLE
Resolve ${workspaceFolder} variable in opa.path setting

### PR DESCRIPTION
**Pull request by johanfylling**
Fixes: #44
Signed-off-by: Johan Fylling <johan.dev@fylling.se>

### CODE REVIEW

**Comment by johanfylling** [comment]
This PR depends on #57, and should be diffed against the changes made there.

File: src/opa.ts [hunk 1]
```diff
@@ -189,7 +189,12 @@ export function run(path: string, args: string[], stdin: string, onSuccess: (std
 // runWithStatus executes the OPA binary at path with args and stdin. The
 // callback is invoked with the exit status, stderr, and stdout buffers.
 export function runWithStatus(path: string, args: string[], stdin: string, cb: (code: number, stderr: string, stdout: string) => void) {
-    const opaPath = vscode.workspace.getConfiguration('opa').get<string>('path');
+    let opaPath = vscode.workspace.getConfiguration('opa').get<string>('path');
+    if (opaPath !== undefined) {
+        opaPath = opaPath.replace('${workspaceFolder}', vscode.workspace.workspaceFolders![0].uri.fsPath.toString())
+    }
+    vscode.Uri.file
```
Comment (srenatus) [inline src/opa.ts:196]: I'm not familiar with the codebase, but this line looks odd. What does it do?

Comment (johanfylling) [reply src/opa.ts:196]: Hum, that looks like some junk left behind by me playing around. Good eyes. Will remove.

**Review by srenatus — APPROVED** [review]
Let's get this merged when rebased and squashed.

**Pull request merged**